cmd/kpod/ps.go

```diff
@@ -302,19 +299,23 @@ func generateContainerFilterFuncs(filter, filterValue string, runtime *libpod.Ru
 }
 
 // generate the template based on conditions given
-func genPsFormat(quiet, size, namespace bool) (format string) {
+func genPsFormat(format string, quiet, size, namespace bool) string {
```

Comment:
This function needs some additional work, since I don't seem to be able to pass multiple options at the same time.
Are these exclusive options?  Do I get and error if I pass --format --quiet --size --namespace together?

Probably not related to this PR, but I think this is a potential bug.